pic-sure-hpds-ui/httpd/httpd-vhosts.conf

```diff
@@ -1,4 +1,3 @@
-Listen 0.0.0.0:80
```

Comment:
This listen is redundant and conflicts with the one in httpd.conf in the latest version